Controller/AjaxController.php

```diff
@@ -156,7 +156,7 @@ protected function datatablesAction(Request $request)
     }
 
     /**
-     * Get date params from session / or set defaults
+     * Get date params and set defaults
```

Comment:
Isn't this method needed by other methods (sourceRevenue) to get the values from session? It appears we're no longer using the session at all...?